katdal/test/test_dataset.py

```diff
@@ -139,10 +139,13 @@ def test_pointing(self):
     def test_uvw(self):
         u, v, w = self.target.uvw(self.antennas[0], self.timestamps,
                                   self.antennas[1])
-        assert_array_equal(self.dataset.u[:, 4], u)
-        assert_array_equal(self.dataset.v[:, 4], v)
-        assert_array_equal(self.dataset.w[:, 4], w)
+        # The answers now differ because the target direction is calculated once
+        # from the array reference instead of from one of the dishes in each
+        # correlation product pair (the whole reason for the speed-up!)
```

Comment:
You could still compute the reference value as the difference between two UVWs measured to the array centre.

Reply:
Good point.

Reply:
Now fixed.